tests/h/presenters/group_json_test.py

```diff
@@ -129,6 +130,47 @@ def test_expanded_organizations_None_if_missing(
 
         assert model["organization"] is None
 
+    def test_it_expands_scopes(self, factories, GroupContext):
+        group = factories.OpenGroup(
+            enforce_scope=False,
+            scopes=[
+                factories.GroupScope(origin="http://foo.com"),
+                factories.GroupScope(origin="https://foo.com"),
+            ],
+        )
+        group_context = GroupContext(group)
+        presenter = GroupJSONPresenter(group_context)
+
+        model = presenter.asdict(expand=["scopes"])
+
+        assert "scopes" in model
+        assert model["scopes"]["enforced"] is False
+        assert set(model["scopes"]["uri_patterns"]) == set(
+            ["http://foo.com*", "https://foo.com*"]
+        )
+
+    def test_expanded_scopes_uri_patterns_empty_if_no_scopes(
+        self, factories, GroupContext
+    ):
```

Comment:
I leave it to you to make a judgement call, but this particular test strikes me as something that could be covered fine at just the unit test level. If the previous test works, and we have unit tests covering this case, it seems very unlikely that it would fail here.

Reply:
Derp. This was actually a holdover from a point in the process where I had a different logic for generating the represented object. It's no longer very relevant. I'll remove.